src/geojson/geojson.ml

```diff
@@ -315,12 +350,22 @@ module Make (J : Geojson_intf.Json) = struct
         | Some typ -> (
             match J.to_string typ with
             | Ok "FeatureCollection" -> (
-                match J.find json [ "features" ] with
-                | Some features ->
-                    J.to_list
+                match (J.find json [ "features" ], J.find json [ "bbox" ]) with
```

Comment:
Since we keep the bounding box as an optional type we don't need two different cases for this

```ocaml
match (J.find json [ "features" ], J.find json [ "bbox" ]) with
  | Some features, bbox ->
      let bbox = Option.map (decode_or_err J.to_array (decode_or_err J.to_float)) bbox in 
      match (J.to_list (fun geometry -> decode_or_err of_json geometry) features) with 
        | Ok v -> Ok (v, None)
        | Error e -> Error e
```

There's maybe an argument to be made that multiple cases is more efficient but I think it would be pretty negligible :))